src/nft/utils/listNfts.ts

```diff
@@ -239,14 +241,14 @@ export async function signListing(
         price: parseEther(listingPrice.toString()),
         tokens: [
           {
-            token: asset.asset_contract.address,
+            token: asset.asset_contract.address ?? '',
```

Comment:
feels odd to have an empty string here be a token address 🤔 what's the expected behavior?

Reply:
I basically just wanted the listing to break as we shouldn't be able to list without a tokenId or collectionAddress. I've moved the check for this earlier in the function